### PR TITLE
The `--api-servers` is no longer supported by Kubelet 1.9, use the `kubelet.kubeconfig` approach.

### PR DESCRIPTION
**Pull request by adelton**
Addressing (partially) https://github.com/kubernetes/website/issues/7521 and https://bugzilla.redhat.com/show_bug.cgi?id=1549151.

### CODE REVIEW

File: init/systemd/environ/kubelet [hunk 1]
```diff
@@ -10,5 +10,9 @@ KUBELET_ADDRESS="--address=127.0.0.1"
 # You may leave this blank to use the actual hostname
 KUBELET_HOSTNAME="--hostname-override=127.0.0.1"
 
+# Edit the kubelet.kubeconfig to have correct cluster server address
+# then uncomment this line
+# KUBELET_KUBECONFIG=/etc/kubernetes/kubelet.kubeconfig
```
[the remark this answers is not in the document]
Comment (adelton) [reply init/systemd/environ/kubelet]: True.

File: init/systemd/kubelet.service [hunk 2]
```diff
@@ -11,7 +11,7 @@ EnvironmentFile=-/etc/kubernetes/kubelet
 ExecStart=/usr/bin/kubelet \
 	    $KUBE_LOGTOSTDERR \
 	    $KUBE_LOG_LEVEL \
-	    $KUBELET_API_SERVER \
+	    $KUBELET_KUBECONFIG \
```
[the remark this answers is not in the document]
Comment (adelton) [reply init/systemd/kubelet.service]: Undefined value should not harm. We can of course uncomment that `KUBELET_KUBECONFIG` assignment above if you prefer.

**Comment by adelton** [comment]
> This fix is applicable only for 1-node cluster deployments.

Well, it's an attempt to address https://github.com/kubernetes/website/issues/7521 which is about https://kubernetes.io/docs/getting-started-guides/fedora/fedora_manual_config/ which is specifically single-node setup.

Would the proposed changes (if they get updated to address your comments) harm the multi-host setup, compared to the state today?

**Comment by adelton** [comment]
Rebased on master, uncommented `KUBELET_KUBECONFIG`, changed the `server` to `http://127.0.0.1:8080/` -> f4ce29dd35b68f538a5845d7e294bbf056d5d215.

**Comment by ingvagabund** [comment]
None of the changed files are used during the Ansible deployment. Given the systemd files are supposed to be used with the latest k8s version, they are currently unusable. So it's safe to merge the changes. Assuming admin/user will modify the files properly.

Thanks Jan for the fixes.

**Pull request merged**